css/builder.css

```diff
@@ -1304,3 +1304,34 @@ form hr {
     display: flex !important;
   }
 }
+
+[data-widget-package="com.fliplet.form-builder"]
+  .form-group.fl-time-picker.time-picker-start
+  > .form-control,
+[data-widget-package="com.fliplet.form-builder"]
+  .form-group.fl-time-picker.time-picker-end
+  > .form-control {
+  padding-right: 0 !important;
```

Comment:
@armine-fliplet Is `!important` needed if `[data-widget-package="com.fliplet.form-builder"]` is already adding more specificity?

Reply:
@tonytlwu nope, it's not needed